### PR TITLE
feat: add support for badger2 as a P2P store + add orch init command

### PR DESCRIPTION
**Pull request by SweeXordious**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: cmd/qgb/orchestrator/cmd.go [hunk 1]
```diff
@@ -134,10 +177,41 @@ func Command() *cobra.Command {
...
 			orch.Start(ctx)
 
 			return nil
 		},
 	}
 	return addOrchestratorFlags(command)
 }
+
+// Init initializes the orchestrator store and creates necessary files.
+func Init() *cobra.Command {
```
Comment (SweeXordious) [inline cmd/qgb/orchestrator/cmd.go:190]: Now that we have the orchestrator starting in 2 steps: init then start. Then, we should update the dockerfile accordingly.

Comment (SweeXordious) [reply cmd/qgb/orchestrator/cmd.go:190]: https://github.com/celestiaorg/orchestrator-relayer/issues/253

**Comment by SweeXordious** [comment]
CI will be fixed by this: https://github.com/celestiaorg/orchestrator-relayer/pull/256

File: store/errors.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package store
+
+import "errors"
+
+var (
+	// ErrOpened is thrown on attempt to open already open/in-use Store.
+	ErrOpened = errors.New("node: store is in use")
```
Comment (SweeXordious) [inline store/errors.go:7]:
```suggestion
	ErrOpened = errors.New("store is in use")
```

File: store/errors.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+	// ErrNotInited is thrown on attempt to open Store without initialization.
+	ErrNotInited = errors.New("node: store is not initialized")
```
Comment (SweeXordious) [inline store/errors.go:9]:
```suggestion
	ErrNotInited = errors.New("store is not initialized")
```

File: store/fslock/lock_unix.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
+//go:build darwin || freebsd || linux
```
Comment (SweeXordious) [inline store/fslock/lock_unix.go:1]:
```suggestion
```

File: store/fslock/lock_unix.go [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (SweeXordious) [inline store/fslock/lock_unix.go:2]:
```suggestion
```

**Review by evan-forbes — DISMISSED** [review]

File: store/fslock/lock_unix.go [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+package fslock
```
Comment (SweeXordious) [inline store/fslock/lock_unix.go:3]: implementation taken from here https://github.com/celestiaorg/celestia-node/tree/main/libs/fslock

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**